lib/msal-common/test/request/AuthenticationHeaderParser.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { AuthenticationHeaderParser } from "../../src/request/AuthenticationHeaderParser";
+import { TEST_AUTHENTICATION_HEADERS, TEST_POP_VALUES } from "../test_kit/StringConstants";
+import { HeaderNames } from "../../src/utils/Constants";
+import { ClientConfigurationError } from "../../src";
+import "isomorphic-fetch";
```

Comment:
Is this used here?

Reply:
Yes, it's required to use the `Headers` class in the tests

Reply:
Removed the import and used `NetworkRequestOptions` / Record<string, string> type instead of Fetch Headers